config-provisioning/src/main/java/com/yahoo/config/provision/Environment.java

```diff
@@ -53,14 +53,7 @@ public static Environment from(String value) {
 
     /** Returns a name of this which is used in external API's and stored in persistent stores */
     public String value() {
-        switch(this) {
-            case prod : return "prod";
```

Comment:
The reason we do this is to have an explicit serialized form, so that renaming an identifier doesn't unintentionally break serialization or change APIs (as the javadoc clearly states).

Same for `SystemName`.

Reply:
Reverted back to previous code and added `vaas` case explicitly.